ensime_shared/vim_commands.py

```diff
@@ -0,0 +1,44 @@
+commands = {
```

Comment:
Not sure this is a great idea. People usually want to have this in the same file where all the logic is. Also, it allows future developers to realise that they shouldn't embed any vim command in the code, but factor it out.